docs/api/covidcast-signals/safegraph.md

```diff
@@ -47,3 +50,39 @@ size. The same method is used for aggregation to states.
 SafeGraph provides this data with a three-day lag, meaning estimates for a
 specific day are only available three days later. It may take up to an
 additional day for SafeGraph's data to be ingested into the COVIDcast API.
+
+
+# SafeGraph Weekly Patterns
+
+* **Source name:** `safegraph`
+* **Number of data revisions since 23 June 2020:** 0
+* **Date of last change:** never
+* **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
+
+Data source based on
+[Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) dataset. SafeGraph provides this data for
+different points of interest ([POIs](https://docs.safegraph.com/v4.0/docs#section-core-places)) considering individual census block groups, using differential privacy to protect individual people's privacy in data.
+
+Delphi gathers the number of daily visits to POIs of certain types(bars, restaurants, etc.) 
+from SafeGraph Weekly Patterns data at the 5-digit ZipCode level, then aggregates and reports these features to the county, MSA, HRR, and state levels. The aggregated data is freely available through the COVIDcast API.
```

Comment:
Should probably say "SafeGraph's Weekly Patterns data". I am assuming "Weekly Patterns" belongs to SafeGraph in this sentence based on how it is used. If I am incorrect and the whole thing is simply a title scratch this comment.